src/anonymous-credentials/schema.ts

```diff
@@ -402,14 +402,15 @@ export interface IJsonSchema {
 }
 
 export interface ISchemaParsingOpts {
+  /** Whether to use the default values or throw error if required parameters are not passed */
   useDefaults: boolean;
   defaultMinimumInteger: number;
   defaultMinimumDate: number;
   defaultDecimalPlaces: number;
 }
 
 export const DefaultSchemaParsingOpts: ISchemaParsingOpts = {
-  useDefaults: false,
+   useDefaults: false,
```

Comment:
Is the space here at the beginning of the line intended?

Reply:
No.